about.css

```diff
@@ -0,0 +1,28 @@
+body
+{
+    background-color: rgb(218, 100, 100);
+    color: white;
+}
+
+.header
+{
+    display: flex;
+    justify-content: space-between;
+    color: white;
+    /* align-items: center; */
+    margin: 5px;
+    /* margin-bottom: 10px; */
+    /* justify-content: space-around; */
```

Comment:
Delete commented out rules. If you were having issues with flexbox here just lmk and I'll see if I can help!